portal/gil/templates/gil/base.html

```diff
@@ -44,6 +44,23 @@
         {{_("Loading")}}... &nbsp;
   </div>
   <section id="mainHolder">
+    <input type="hidden" id="currentUserId" value="{{user.id if user}}" />
+    {% if user and (user.has_role(ROLE.PATIENT) or user.has_role(ROLE.PARTNER)) %}
+      <input type="hidden" id="interventionMenu" value="true" />
+    {% endif %}
+    {% if timed_out %}
+      <input type="hidden" id="sessionTimedOut" value="true" />
+    {% endif %}
+    {% if request.args.get("disableLinks", "") %}
+      <input type="hidden" id="disableLinks" value="true" />
+    {% endif %}
+    {% if user and user.has_role(ROLE.CONTENT_MANAGER) %}
+      <input type="hidden" id="isContentManager" value="true" />
+    {% endif %}
+    <input type="hidden" id="portalURI" value="{{PORTAL}}" />
+    <input type="hidden" id="csrfToken" value="{{csrf_token()}}" />
+    <input type="hidden" id="env" value="{{config.SYSTEM_TYPE if config.SYSTEM_TYPE}}" />
```

Comment:
Same suggestion as above, it might be nice to query and persist these config values on the frontend